frontend/src/components/supp-allowances/SupportNeedsSummary.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-container fluid class="pa-2 pb-0">
+    <div class="mt-4">
+      <h4 class="text-decoration-underline">Funding Selected</h4>
+      <ul>
+        <li v-for="item in selectedFunding" :key="item.value" class="ml-8 my-2">
+          {{ item.label }}
+        </li>
+      </ul>
+    </div>
+
+    <div v-if="isOtherBoxDisplayed">
+      <p v-if="supportModel.supportOtherDescription" class="ml-12 my-2">
+        {{ supportModel.supportOtherDescription }}
+      </p>
+      <p v-else>
+        <AppMissingInfoError :to="{ name: 'supp-allowances-form', params: { applicationGuid: $route.params.applicationGuid } }">
+          {{ APPLICATION_ERROR_MESSAGES.OTHER }}
+        </AppMissingInfoError>
+      </p>
+    </div>
+  </v-container>
+</template>
+
+<script>
+import AppMissingInfoError from '@/components/ui/AppMissingInfoError.vue'
+import { APPLICATION_ERROR_MESSAGES } from '@/utils/constants'
+import { SUPPORT_CHECKBOX_LABELS } from './suppConstants'
+
+export default {
+  components: { AppMissingInfoError },
+  props: {
+    supportModel: {
+      type: Object,
```

Comment:
consider if supportModel should be required: true

Reply:
done